src/main/kotlin/me/partlysanestudios/partlysaneskies/config/keybinds/RefreshKeybinds.kt

```diff
@@ -24,7 +24,7 @@ import net.minecraftforge.client.event.GuiScreenEvent
 import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
 import org.lwjgl.input.Keyboard
 
-class RefreshKeybinds {
+object RefreshKeybinds {
```

Comment:
I also did some code clean up


(please can we use objects instead of classes with single instances